settings.ini

```diff
@@ -0,0 +1,15 @@
+[versions]
+FIRMWARE_VERSION=2021.11.22.0-3
+DIAGNOSTICS_VERSION=625ea72
+CONFIG_VERSION=8cc544f
+PKTFWD_VERSION=8d8de7d
+MINER_VERSION=d760384
+UPNP_VERSION=b575a2f
+
+[device_rpi]
+ARCH=arm64
+I2C_DEVICE=/dev/i2c-1
+
+[device_rockpi]
+ARCH=arm64
+I2C_DEVICE=/dev/i2c-7
```

Comment:
Not sure the best way to handle this, but we already have the i2c bus in hm-pyhelper so perhaps we should instead pull this from there?

Perhaps the arch could be stored there as well so that the settings.ini here is only used for versions that are specific to this repo and not better dealt with elsewhere.

Reply:
Having said that, perhaps that additional work could be pushed into another issue / PR

Reply:
created #336